SpringBootExample/pom.xml

```diff
@@ -7,7 +7,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.5.12</version>
+		<version>2.7.3</version>
```

Comment:
Does line 15 (below) also need to be updated? Looks like it's got a spring-boot.version value too.

Reply:
properties defined in pom are like placeholders. Since versions are hardcoded, this line is never used. But it's a good practice to start using these placeholders. For now I have updated the version here too.